internal/factsengine/gatherers/ini_files_test.go

```diff
@@ -0,0 +1,287 @@
+package gatherers_test
+
+import (
+	"context"
+	"testing"
+
+	st "github.com/balanza/supertouch"
```

Comment:
question: why did you choose to create your own project for this?
afero was not good enough?
I think we could have a tiny presentation about the  package to the rest of the team to avoid isolated knowledge

Reply:
I wished `afero` could do it, but it does not afaik.

That's a utility I made for another project, I kinda missed it here. We may as well copy the files, it's not  rocket science

Reply:
We already use `afero` to create on memory file systems. For example: https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sapservices_test.go#L39
Actually, we chose to use `afero` to make testing possible

Reply:
Fair enough, I removed it in https://github.com/trento-project/agent/pull/395/commits/5a1d8cefbd7570b23e7e2da52a3193138081efd2